<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TravelAgencyUI</class>
 <widget class="QMainWindow" name="TravelAgencyUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>704</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TravelAgencyUI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_Kunde">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>10</y>
      <width>581</width>
      <height>281</height>
     </rect>
    </property>
    <property name="title">
     <string>Kund*in</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ID</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_ID">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>31</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Reisen</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Name">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>70</y>
       <width>181</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget_Reisen">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>100</y>
       <width>401</width>
       <height>171</height>
      </rect>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderHighlightSections">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Reise-ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Beginn der Reise</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Ende der Reise</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_Reise">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>581</width>
      <height>301</height>
     </rect>
    </property>
    <property name="title">
     <string>Reise</string>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Reise-ID</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_ReiseID">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>41</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Buchungen</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget_Buchungen">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>90</y>
       <width>481</width>
       <height>192</height>
      </rect>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Buchungstyp</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Start</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Ende</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Preis</string>
      </property>
     </column>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>602</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatei">
    <property name="title">
     <string>Datei</string>
    </property>
    <addaction name="actionDatei_einlesen"/>
   </widget>
   <widget class="QMenu" name="menuKund_Innen">
    <property name="title">
     <string>Kund*Innen</string>
    </property>
    <addaction name="actionSuche"/>
   </widget>
   <widget class="QMenu" name="menuSpeichern">
    <property name="title">
     <string>Speichern</string>
    </property>
    <addaction name="action_iatacodes_einlesen"/>
    <addaction name="actionSpeichern"/>
   </widget>
   <widget class="QMenu" name="menu_iatacodes">
    <property name="title">
     <string> iatacodes</string>
    </property>
    <addaction name="action_iatacodes_einlesen"/>
   </widget>
   <addaction name="menuDatei"/>
   <addaction name="menuKund_Innen"/>
   <addaction name="menuSpeichern"/>
   <addaction name="menu_iatacodes"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionDatei_einlesen"/>
   <addaction name="actionSuche"/>
   <addaction name="actionSpeichern"/>
   <addaction name="action_iatacodes_einlesen"/>
  </widget>
  <action name="actionDatei_einlesen">
   <property name="icon">
    <iconset>
     <normaloff>import_icon.png</normaloff>import_icon.png</iconset>
   </property>
   <property name="text">
    <string>Datei einlesen</string>
   </property>
  </action>
  <action name="actionSuche">
   <property name="icon">
    <iconset>
     <normaloff>search.png</normaloff>search.png</iconset>
   </property>
   <property name="text">
    <string>Suche</string>
   </property>
  </action>
  <action name="actionSpeichern">
   <property name="icon">
    <iconset>
     <normaloff>speichern.jpg</normaloff>speichern.jpg</iconset>
   </property>
   <property name="text">
    <string>Speichern</string>
   </property>
  </action>
  <action name="action_iatacodes_einlesen">
   <property name="icon">
    <iconset>
     <normaloff>iatacodes.jpeg</normaloff>iatacodes.jpeg</iconset>
   </property>
   <property name="text">
    <string> iatacodes einlesen</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
